src/argilla_llama_index/llama_index_handler.py

```diff
@@ -124,81 +77,40 @@ def __init__(
         if api_key is None and os.getenv("ARGILLA_API_KEY") is None:
             warnings.warn(
                 (
-                    "Since `api_key` is None, and the env var `ARGILLA_API_KEY` is not"
+                    "Since `api_key` is None, and the environment var `ARGILLA_API_KEY` is not"
                     f" set, it will default to `{DEFAULT_API_KEY}`, which is the"
                     " default API key in Argilla Quickstart."
                 ),
             )
             api_key = DEFAULT_API_KEY
-    
-        # Connect to Argilla with the provided credentials, if applicable
+
+        # Initialize the Argilla client
         try:
```

Comment:
This logic should also be handled internally. 

I think we should be able to pass the `api_key` and `api_url` directly to the `rg.init` and after handle all logic in the same was as the SDK normally does that.